rotkehlchen/db/dbhandler.py

```diff
@@ -1320,21 +1320,18 @@ def remove_blockchain_accounts(
                 f'{blockchain.value} accounts that do not exist',
             )
 
-    def _get_address_details_if_time(
+    def _get_address_details(
             self,
             cursor: 'DBCursor',
             address: ChecksumEthAddress,
-            current_time: Timestamp,
     ) -> Optional[Dict[str, Any]]:
         query = cursor.execute(
-            'SELECT tokens_list, time FROM ethereum_accounts_details WHERE account = ?',
+            'SELECT tokens_list FROM ethereum_accounts_details WHERE account = ?',
```

Comment:
If you code it this way then the `time` argument is no longer needed. I think it should be. And we should even show to the user for each address when is the last time tokens were asked to be detected.

Remember this is only for the "brute-force" detection where we essentially iterate every token we know. This is always gonna be slow.

The other nice type of detection we will do is process all transaction events and there we can see all tokens to touch an address. We can even (in theory) calculate balances this way.

Reply:
Yeah, added time to the response